docs/management/connectors/action-types/email.asciidoc

```diff
@@ -228,6 +228,10 @@ is considered `false`.  Typically, `port: 465` uses `secure: true`, and
 [[elasticcloud]]
 ==== Sending email from Elastic Cloud
 
+IMPORTANT: You need to 
+  link:{cloud}/ec-watcher.html#ec-watcher-whitelist[allowlist] the email 
+  addresses that notifications get sent.
```

Comment:
Should this get `.. that notifications get sent to` ?

Reply:
I think the two sentences have the same meaning but the first one is shorter and `sent to` feels like a colloquialism for my non-native English mind.

Reply:
Words are fun! :) Here are some other options to consider:

- To receive notifications, the email addresses must be added to an allowlist in the Elasticsearch Service Console.
- You must allowlist the email addresses; otherwise they are not sent notifications(? or do they just error out?).

Reply:
Updated in [05dc6c8](https://github.com/elastic/kibana/pull/134552/commits/05dc6c8945cf52b3ac7a9b5f8d2a8505896e99f8).